archive/r/rust/capitalize.rs

```diff
@@ -0,0 +1,23 @@
+// Accept a string from User and Capitalize the first letter of that string
+// Rust is built to support not just ASCII, but Unicode by-default.
+// Do note, each character in string is multi-byte UTF-8 supported, which needs upto 3 bytes (to accommodate Japanese letters)
+// Best part of Rust compiler issues warning to remove unused variables, functions, ...
+fn main() {
+    // Use Std. Input Device
+    use std::io::{stdin};
+    // Create a new String Object
+    let mut input_value = String::new();
+    // Read a mutable input value from Std Input with newline at the end. If it fails, throw Error Message
+    stdin().read_line(&mut input_value).expect("Error: reading from keyboard failed");
```

Comment:
Instead of using the read_line command, we's prefer using the main arguments (command line).

Reply:
I have updated the program to accept string from commandline.